src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -2962,6 +2964,7 @@ def test_sql_elastic_pools_vcore_mgmt(self, resource_group, resource_group_locat
     @ResourceGroupPreparer(name_prefix='clitest-HSEP', location='eastus2')
     @SqlServerPreparer(name_prefix='clitest-HSEP', location='eastus2')
     @AllowLargeResponse()
+    @live_only() # Could not find tier Hyperscale. Supported tiers are: ['Standard', 'Premium', 'Basic', 'GeneralPurpose', 'BusinessCritical']
```

Comment:
Is this a breaking change that `Hyperscale` not supported any more in `azure-mgmt-sql==4.0.0b4`?

Reply:
I spoke with engineer who implemented this, and she said that Hyperscale type is still not publicly available. She confirmed that I can skip the test